recon/src/recon/tests.rs

```diff
@@ -1810,97 +1735,141 @@ async fn alternating() {
             dog: [a: A, c: C, d: D, f: F, h: H, j: J, l: L, n: N, p: P, q: Q, s: S, u: U, w: W, y: Y, z: Z]
         -> range_req({𝚨 h(a, b, c, e, g, i, k, m, o, p, r, t, v, x, z)#15 𝛀 })
             cat: [a: A, b: B, c: C, e: E, g: G, i: I, k: K, m: M, o: O, p: P, r: R, t: T, v: V, x: X, z: Z]
-        <- range_resp({𝚨 h(a, c, d, f, h, j, l)#7 n}, {n h(p, q, s, u, w, y, z)#7 𝛀 })
+        <- range_resp({𝚨 h(a, c, d, f, h, j, l)#7 n}, {n h(n, p, q, s, u, w, y, z)#8 𝛀 })
```

Comment:
This test (`alternating`) seems to have been hit hardest in terms of round trip increases but most stayed the same or dropped a bit (e.g. `subset_interest`).

Reply:
I had a shower thought on how I might reduce roundtrips. I may try it out today...